src/Tempest/Discovery/DiscoveryDiscovery.php

```diff
@@ -42,7 +51,20 @@ public function storeCache(): void
 
     public function restoreCache(Container $container): void
     {
-        $discoveryClasses = unserialize(file_get_contents(self::CACHE_PATH));
+        $discoveryClasses = unserialize(file_get_contents(self::CACHE_PATH), [
+            'allowed_classes' => [
+                CommandBusDiscovery::class,
```

Comment:
I believe these are all the discovery classes there are.

Reply:
Hmm yeah this will be annoying, since packages can provide their own discovery classes. This special case will need another solution.